neurolib/models/fhn/model.py

```diff
@@ -15,8 +15,8 @@ class FHNModel(Model):
     state_vars = ["x", "y", "x_ou", "y_ou"]
     output_vars = ["x", "y"]
     default_output = "x"
-    input_vars = ["x_ext", "y_ext"]
-    default_input = "x_ext"
```

Comment:
These variables are for external stimulation and therefore they should be left as is.